test/loadEnvironment.test.ts

```diff
@@ -0,0 +1,104 @@
+import * as fs from "fs"
+import { loadEnvironment } from "../src/loadEnvironment"
+
+const getContentfulEnvironmentFileFactory = jest.fn((_type: string) => {
```

Comment:
I'd say if we don't need he `_type` argument we could get rid of it, right?

```suggestion
const getContentfulEnvironmentFileFactory = jest.fn(() => {
```

Reply:
Sorry I get it now, it's used at the last assertions